src/Components/ServiceScene/LogsListScene.tsx

```diff
@@ -92,6 +92,12 @@ export class LogsListScene extends SceneObjectBase<LogsListSceneState> {
     }
   }
 
+  clearSelectedLine() {
+    this.setState({
+      selectedLine: undefined,
+    });
+  }
+
```

Comment:
Is this being called from anywhere?

Reply:
yeah from the LogsTableScene https://github.com/grafana/explore-logs/pull/568/files#diff-30db4d73ccca81096b38ec2e68d2c0ac8969ddf5c062c0ef01ba2dfbcebb1c0bR40-R45